data/map systems.txt

```diff
@@ -24059,6 +25463,25 @@ system Nocte
 		distance 2260.97
 		period 3701.13
 
+system Noitakt
+	pos -129.13889 225.41667
+	government Syndicate
+	link Mirach
+	link Schedar
+	asteroids "small rock" 4 2.604
+	asteroids "large rock" 7 3.069
+	asteroids "small metal" 9 3.069
+	asteroids "medium metal" 9 4.1385
+	asteroids "large metal" 2 4.278
+	minables aluminum 1 5.7519675
+	minables lead 7 6.190245
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
+	habitable 3276.8
+	belt 1478
```

Comment:
```suggestion
	habitable 176.8
	belt 1478
	object
		sprite star/g-dwarf
		distance 5.1
		period 11.3
	object
		sprite star/l-dwarf
		distance 33.2
		period 11.3
	object
		sprite planet/rock13
		distance 1564.3
		period 476.772
	object
		sprite planet/rock11
		distance 2811.2
		period 1288.23
		object
			sprite planet/ice7
			distance 211
			period 12.901
```

System objects. Went for a relatively empty style, based on EF Eridani - the G dwarf is a formerly accreting white dwarf, while the L dwarf is a stripped stellar remnant too small to continue fusion. No planets in close orbits are left, because they would have been destroyed during the red giant phase of the G dwarf.